robot_msgs/PointCloud cloud
---
string object
robot_msgs/PoseStamped pose
